### DISCUSSION TITLE
Paragraphs is missing "speaker" and "num_words" property in .NET SDK

### DISCUSSION THREAD

**Discussion by jooooel** [deepgram/community#836] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

I'm using the .NET SDK (nuget version 4.0.2) to transcribe some audio. When deserializing the transcription, it misses the `speaker` and `num_words` properties on the `paragraph`level.

Here are two outputs. The first is json from a curl request and the second is what I get if I serialize the .NET response:

```
From curl
"paragraphs": [
  {
    "sentences": [
      {
        "text": "So I think Marcus is joining us soon.",
        "start": 2.6399999,
        "end": 4.8199997
      },
      { "text": "Yeah.", "start": 4.88, "end": 5.38 }
    ],
    "speaker": 0,
    "num_words": 9,
    "start": 2.6399999,
    "end": 5.38

// From .NET
"paragraphs": [
  {
    "sentences": [
      {
        "text": "So I think Marcus is joining us soon.",
        "start": 2.6399999,
        "end": 4.8199997
      },
      { "text": "Yeah.", "start": 4.88, "end": 5.38 }
    ],
    "start": 2.6399999,
    "end": 5.38
  },
```

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

curl -X POST -H "Authorization: Token SECRET" -H "Content-Type: application/json" -d '{"url":"my-url"}' "https://api.deepgram.com/v1/listen?smart_format=true&punctuate=true&paragraphs=true&diarize=true&language=en&model=nova-2"

### If possible, please attach your code or paste it into the text box.

```
curl -X POST -H "Authorization: Token SECRET" -H "Content-Type: application/json" -d '{"url":"my-url"}' "https://api.deepgram.com/v1/listen?smart_format=true&punctuate=true&paragraphs=true&diarize=true&language=en&model=nova-2"
```

```
        Library.Initialize();

        var deepgramClient = new PreRecordedClient("SECRET");

        var response = await deepgramClient.TranscribeUrl(
            new UrlSource(myUrl),
            new PreRecordedSchema()
            {
                Model = "nova-2",
                Language = "en",
                SmartFormat = true, 
                Punctuate = true, 
                Paragraphs = true, 
                Diarize = true,
            });

        Console.WriteLine(JsonSerializer.Serialize(
            response,
            new JsonSerializerOptions() { WriteIndented = true }
        ));

        Library.Terminate();
```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
@jooooel Thank you for reporting this. I also see this happening, which means it is a bug. I have created an issue in the .NET SDK here: https://github.com/deepgram/deepgram-dotnet-sdk/issues/313

We will aim to get a fix out by the end of the week. Thanks!

**Comment by SandraRodgers** [comment]
@jooooel This has been fixed. You can update to the most recent version of the SDK to get that fix.
